utils/spdx/src/main/kotlin/SpdxConstants.kt

```diff
@@ -70,6 +70,11 @@ object SpdxConstants {
      */
     const val LICENSE_LIST_URL = "https://spdx.org/licenses/"
 
+    /**
+     * The package verification code for no input.
+     */
+    const val EMPTY_PACKAGE_VERIFICATION_CODE = "da39a3ee5e6b4b0d3255bfef95601890afd80709"
```

Comment:
The constant is only used by tests. What's the reason for putting it into production code?

Reply:
The test verifies the correctness of the constant. The idea is to later use the constant in production code where (for whatever reason) having a package verification code is enforced, but there was no actual data to compute it on.